get_data.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
+
+$username = 'homestead';
+$password = 'secret';
+
+$easy_db = \ParagonIE\EasyDB\Factory::create(
+  'pgsql:host=localhost;dbname=homestead',
+  $username,
+  $password
+);
+
+$socrata = new Socrata('data.medicare.gov');
+//$hospitals = $socrata->get('/resource/rbry-mqwu.json?$limit=10000000');
+
+$bad_ones = $socrata->get('/resource/5u3e-gkdf.json?$limit=100000000');
```

Comment:
note the `?$limit={{n}}`

SOCRATA limits results to 1000 by default. Need to up that if you want everything all at once